src/ucp/wireup/address.c

```diff
@@ -613,6 +613,7 @@ static ucs_status_t ucp_address_do_pack(ucp_worker_h worker, ucp_ep_h ep,
         ptr             = UCS_PTR_TYPE_OFFSET(ptr, worker->uuid);
     }
 
+    worker->context->config.features |= UCP_FEATURE_RMA;
```

Comment:
This is required to prevent the assertion failure in 
ucp_rkey.c:118  UCX  ERROR feature flags UCP_FEATURE_RMA,UCP_FEATURE_AMO32,UCP_FEATURE_AMO64 were not set for ucp_init()